sympy/core/assumptions.py

```diff
@@ -173,7 +173,7 @@
     'zero           ->  even & finite',
 
     'prime          ->  integer & positive',
-    'composite      ==  integer & positive & !prime',
+    'composite      ->  integer & positive & !prime',
```

Comment:
Before committing I am thinking more about this...so this means if something is integer & positive and !prime that it won't be composite:

```
>>> var('i', integer=1, positive=1, prime=False)
i
>>> i.is_composite
```

I don't like having 1 being composite, but I also don't like the above being None. Is there any way to remedy that?